Base/usr/share/man/man5/font.md

```diff
@@ -1,17 +1,40 @@
 ## Name
 
-font - Bitmap Font File format
+font - Bitmap Font File format (.font)
 
 ## Synopsis
 
-The .font file format stores bitmap fonts in SerenityOS's own binary format.
+Font files contain bitmap definitions of fonts (Gfx::BitmapFont).
 
 ## Description
 
-These files contain bitmap definitions of fonts, either varying-width or fixed-width.
+Bitmap fonts can be either varying-width or fixed-width.
+The first four bytes of font files contain the filemagic: !Fnt (0x21466e74).
 
-The first four bytes contain the filemagic: `!Fnt` (0x21466e74).
+In addition, Gfx::BitmapFont supports reading from and writing to font files (as well as reading directly from memory)
+and the question mark '?' used as a fallback for unknown glyphs or emojis.
+
+## Structure
+
+The order is big-endian.
+
+| size       | member name         |
+|------------|---------------------|
+| [4 bytes]  | filemagic           |
+| [1 byte]   | glyph width         |
+| [1 byte]   | glyph height        |
+| [2 bytes]  | range mask size     |
+| [1 byte]   | variable width flag |
+| [1 byte]   | glyph spacing       |
+| [1 byte]   | baseline            |
+| [1 byte]   | mean line           |
+| [1 byte]   | presentation size   |
+| [2 bytes]  | weight              |
+| [1 byte]   | slope               |
+| [32 bytes] | name                |
+| [32 bytes] | family              |
```

Comment:
Why are all of these in brackets? That's a reference-style link in some markdown flavours, let's not do that

Everything on the right should start with a capital letter